client/flutter/lib/components/page_scaffold/page_header.dart

```diff
@@ -5,44 +5,55 @@ import 'package:auto_size_text/auto_size_text.dart';
 class PageHeader extends StatelessWidget {
   final String title;
   final String heroTag;
+  final Color borderColor;
+  final double fontSize;
```

Comment:
once #1151 is merged, let's add `TypographyVariant` and then maybe make this more generic with a `TextStyle` instead of only allowing font size override?

```suggestion
  final FontStyle titleStyle;
  final TypographyVariant titleVariant;
```

Reply:
Fixed the size -> style issue for now.  Depending on the merge order, will do variant too.